api/src/main/java/org/openmrs/module/locationbasedaccess/LocationBasedAccessActivator.java

```diff
@@ -20,6 +23,8 @@
 
 
     protected Logger log = LoggerFactory.getLogger(getClass());
+    private final String ACCESSLOCATION_UUID = "8b5c95ef-103c-41bc-9f24-368b8f77e070";
```

Comment:
It will be more readable to separate access and location

Reply:
And by the way, what does access location mean, in the first place?

Reply:
Separated